src/pages/RoversPage.scss

```
@@ -0,0 +1,15 @@
+@use "../styles/_variables.scss" as *;
+
+h1 {
+  font-family: $dashboard-font;
+  color: $color-primary-darker;
+  text-align: center;
+  font: size 1.875em;
```

I think this is a typo, and you want instead:
```suggestion
  font-size: 1.875em;
```